src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpMultilineCheck.java

```diff
@@ -104,6 +104,23 @@
  * greedy will allow multiple violations in the example to be found in the same file.
  * </p>
  *
+ * <p>
+ * To configure the check to restrict an empty file:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;RegexpMultiline&quot;&gt;
+ *     &lt;property name=&quot;format&quot; value=&quot;^\s*$&quot; /&gt;
+ *     &lt;property name=&quot;matchAcrossLines&quot; value=&quot;true&quot; /&gt;
+ *     &lt;property name=&quot;message&quot; value=&quot;Empty file is not allowed&quot; /&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>
+ * Example of violation from the above config:
+ * </p>
+ * <pre>
+ * [...This is an empty file] // violation
```

Comment:
there is no ease way to show that file is empty.
please do it as I did at issue description by printing of its content:
```
/var/tmp$ cat Test.java 





/var/tmp$
/var/tmp/Test.java // violation
```
last line it might make sense to put in separate PRE tag, please experiment of better look during rendering.

Reply:
<img width="1256" alt="Screen Shot 2020-04-19 at 8 53 54 PM" src="https://user-images.githubusercontent.com/36587580/79691949-f7ea3400-827f-11ea-97a4-286b56448238.png">
@romani is this okay ?

Reply:
Good